### ISSUE TITLE
Curate Fermi-LAT datasets

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#1827]
We currently have this fail for `tutorials/fermi_lat.ipynb`:
https://travis-ci.org/gammapy/gammapy/jobs/432107192#L1370

What should be done is to put some code here so that the files from the extra repo are added to the data index:
https://github.com/gammapy/gammapy-webpage/blob/1be9538aa590f1cbf9cf1ad1a53c45030d921150/make.py#L65

And then the notebook changed to load all files via `$GAMMAPY_DATA`.

That's a not so much work, but to really curate the Fermi-LAT datasets we have, i.e. reduce the existing `fermi_2fhl`, `fermi_3fhl` and `fermi-survey` to one or two, and to adapt the docs and tests we have to use that, is at least a day or work and would most effectively be done in one go.

So I'll not attempt this now, but open this reminder issue for v0.9.

**Comment by cdeil** [comment]
@adonath - `fermi_lat.ipynb` is broken. We didn't notice because the right dataset isn't put on the build machine executing the notebooks in CI:
```
INFO:__main__:Skipping notebook fermi_lat because requirement is missing.
```
Can you make it better?

https://travis-ci.org/gammapy/gammapy/jobs/479299546#L1438

```
INFO:gammapy.scripts.jupyter:   ... EXECUTING temp/fermi_lat.ipynb
INFO:gammapy.scripts.jupyter:   ... Executing duration: 0.29 mn
INFO:gammapy.scripts.jupyter:   ... FAILED
INFO:gammapy.scripts.jupyter:

TypeError in cell [27]

--SOURCE CODE: 
plt.figure(figsize=(8, 5))

for energy in [100, 300, 1000] * u.GeV:
    psf_at_energy = psf.table_psf_at_energy(energy)
    psf_at_energy.plot_psf_vs_rad(label="PSF @ {:.0f}".format(energy), lw=2)

erange = [50, 2000] * u.GeV
psf_mean = psf.table_psf_in_energy_band(energy_band=erange, spectral_index=2.3)
psf_mean.plot_psf_vs_rad(label="PSF Mean", lw=4, c="k", ls="--")

plt.xlim(1e-3, 0.3)
plt.ylim(1e3, 1e6)
plt.legend();

--TRACEBACK: 
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-27-3e7995db2f82> in <module>
      6 
      7 erange = [50, 2000] * u.GeV
----> 8 psf_mean = psf.table_psf_in_energy_band(energy_band=erange, spectral_index=2.3)
      9 psf_mean.plot_psf_vs_rad(label="PSF Mean", lw=4, c="k", ls="--")
     10 

~/gammapy/gammapy/irf/psf_table.py in table_psf_in_energy_band(self, energy_band, spectrum, n_bins, **kwargs)
    552         psf_value = self.evaluate(energy=energy)
    553         psf_value_weighted = (weights[:, np.newaxis] * psf_value)
--> 554         return TablePSF(self.rad, psf_value_weighted.sum(axis=0), **kwargs)
    555 
    556     def containment_radius(self, energies, fraction, interp_kwargs=None):

TypeError: __init__() got an unexpected keyword argument 'spectral_index'
```

**Comment by adonath** [comment]
Fixed by #1994.
